### DISCUSSION TITLE
error POST https://dc.services.visualstudio.com/v2/track

### DISCUSSION THREAD

**Discussion by Paolo-lab** [iTwin/itwinjs-core#5236] [Q&A]
Hello good morning from this morning I receive a lot of errors on https://dc.services.visualstudio.com/v2/track (error 400)
{"itemsReceived":1,"itemsAccepted":0,"errors":[{"index":0,"statusCode":400,"message":"Invalid instrumentation key"}]}

until yesterday everything was ok but today it doesn't seem to work, it happens to me in the development environment when I zoom in on the model

thanks for the precious help

Paul
<img width="928" alt="errore" src="https://user-images.githubusercontent.com/57446652/225947356-7c0adc91-3516-4078-b7c4-dc672216ec65.PNG">

**Comment by aruniverse** [comment]
Hey Paolo,

You should be able to safely ignore those post failures, used for telemetry.
Mind sharing what version of iTwin.js youre using?
Are you using the iTwin Viewer? If so what version of it are you using?

**Reply by Paolo-lab** [reply]
hi aruniverse , yes of course i can ignore them , but i was wondering why .
my viewer version is "@itwin/web-viewer-react": "^2.0.9"

thanks for the help

Paul

**Reply by aruniverse** [reply]
You might have an ad-blocker installed which blocks posting telemetry. Newer browsers will have this auto enabled by default as well.
In web-viewer-react@3 we removed the telemetry client which was making these requests.
I'd recommend updating to v3, here's our [changelog](https://github.com/iTwin/viewer/blob/master/releases/CHANGELOG-3.0.md)
